spark/sql/types/row.go

```diff
@@ -46,7 +59,7 @@ func (r *rowImpl) Len() int {
 }
 
 func (r *rowImpl) FieldNames() []string {
-	names := make([]string, len(r.offsets))
+	var names []string
```

Comment:
can you fix this line to pickup the latest master changes that pre-allocates please?

Reply:
thanks a lot, that is the best solution, I was confused how to do it 🙏🏽 🙏🏽 🙏🏽 🙏🏽 🙏🏽 

fixing ...

Reply:
done 👍🏽